websites/G/GameBanana/presence.ts

```diff
@@ -0,0 +1,383 @@
+const presence = new Presence({ clientId: "958520351158050887" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let header: HTMLElement;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "gblogo",
+			startTimestamp: browsingTimestamp
+		},
+		sections = [
+			"mods",
+			"scripts",
+			"sounds",
+			"sprays",
+			"tools",
+			"contests",
+			"jams",
+			"concepts",
+			"projects",
+			"requests",
+			"tuts",
+			"wips",
+			"articles",
+			"blogs",
+			"events",
+			"news",
+			"polls",
+			"questions",
+			"threads",
+			"wares",
+			"apps",
+			"bugs",
+			"clubs",
+			"games",
+			"ideas",
+			"models",
+			"positions",
+			"reviews",
+			"studios",
+			"statusupdates",
+			"wikis",
+			"updates",
+			"posts",
+			"support",
+			"stamps",
+			"flags",
+			"permits",
+			"generators",
+			"initiatives"
+		],
+		nogamesections = [
+			"App",
+			"Bug",
+			"Club",
+			"Generator",
+			"Idea",
+			"Initiative",
+			"Model",
+			"Review",
+			"StatusUpdate",
+			"Support",
+			"Studio",
+			"Wiki"
+		];
+
+	// Check if we're on the site
+	if (document.location.hostname === "gamebanana.com") {
+		header = document.querySelector("#PageTitle");
+
+		// Homepage
+		if (!header) presenceData.details = "Viewing the Homepage";
+		// 404
+		else if (header.textContent.includes("Tango Down!"))
+			presenceData.details = "Tango Down!";
+		// Discussions
```

Comment:
You do not need comments when you can infer the page from the pathname on the next line